### PR TITLE
Add azure pipeline

### PR DESCRIPTION
**Pull request by SjurSutterudSagen**

### CODE REVIEW

**Comment by SjurSutterudSagen** [comment]
Need to update https://github.com/statisticsnorway/sagalog-postgres first. Removing the pull request in the mean time.

**Comment by SjurSutterudSagen** [comment]
Sagalog-postgres is migrated. Reopening this now.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,19 +1,69 @@
...
+
+resources:
+  containers:
+    - container:  'ubuntu_container_for_build_to_nexus'
+      image:      'ubuntu:latest'
+  repositories:
+    - repository: templates
+      type:       github
+      name:       statisticsnorway/azure-pipelines-templates
+      ref:        refs/tags/0.0.7
```
Comment (takvamborgen) [inline azure-pipelines.yml:24]: Why not use the latest tag (0.1.2)?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,19 +1,69 @@
...
+
+      # Sonar-Analyze task to check code test coverage
+      - template: maven/task-sonar-analyze.yml@templates
 
-- script: |
-    echo Add other tasks to build, test, and deploy your project.
-    echo See https://aka.ms/yaml
-  displayName: 'Run a multi-line script'
+      # Deploy artifact to Nexus
+      - template: maven/task-install-and-deploy-to-nexus.yml@templates
```
Comment (takvamborgen) [inline azure-pipelines.yml:69]: I believe you can use the task-maven-complete template here

File: azure-pipelines.yml (hunk 1 above)
Comment (SjurSutterudSagen) [reply azure-pipelines.yml:24]: I forgot to update this, fixed now.

**Review by takvamborgen — APPROVED** [review]
Looks good

**Pull request merged**